report/id_list.go

```diff
@@ -1,19 +1,25 @@
 package report
 
-import (
-	"sort"
-)
+import "sort"
 
-// IDList is a list of string IDs, which are always sorted and
-// without duplicates.
+// IDList is a list of string IDs, which are always sorted and unique.
 type IDList []string
 
-// NewIDList makes a new IDList.
-func NewIDList(ids ...string) IDList {
+// MakeIDList makes a new IDList.
+func MakeIDList(ids ...string) IDList {
 	sort.Strings(ids)
 	return IDList(ids)
 }
 
+// Copy returns a value copy, useful for tests.
+func (a IDList) Copy() IDList {
```

Comment:
Is the intention for this to be used by non-test code in the future?  If not, move it into the _test.go file